app/models/notification.rb

```diff
@@ -1,5 +1,5 @@
 class Notification < ApplicationRecord
-  belongs_to :ketchup
-  belongs_to :trip
+  belongs_to :ketchup, optional: true
```

Comment:
what does adding the optional true to the belongs_to here do?

Reply:
if there is no optional true we can't create notification without ketchup and trip id but some notification we need to create without ketchup or trip id for example friend request

Reply:
makes sense